clearwater-infrastructure/etc/bash.bashrc.clearwater

```diff
@@ -1,22 +1,30 @@
+# Read out the node types on the server from the installed packages
 type=$(. /etc/clearwater/config
        {
+         # List all installed packages for CentOS or Debian. rpm returns an unordered list, dpkg-query is already sorted
          if which yum > /dev/null 2>&1 ; then
-           rpm -qa --qf '%{NAME}\n'
+           rpm -qa --qf '%{NAME}\n' | sort | uniq
          else
            dpkg-query -W -f='${Package}\n'
          fi
        } |
-       egrep '^(bono|clearwater-sip-stress|ellis|homer|homestead|sprout|ralf)$' |
+       egrep '^(bono|clearwater-sip-stress|ellis|homer|homestead|sprout|stats-engine|ralf)$' |
+       # Replacements with customer-friendly names and concatenate with hyphens
        sed -e 's/clearwater-sip-stress/sipp/g' |
+       sed -e 's/stats-engine/mvse/g' |
        tr "\\n" "-" |
-       sort |
        sed -e 's/bono-ellis-homer-homestead-sprout/cw-aio/g
                s/-$//g')
+
+# Read out the node index if set
```

Comment:
You're explaining what the code does here rather than why it does it. Better would be:

`# We should only set the node-index if it is configured in the underlying config.`

It might also be worth mentioning why we have chosen to write the code this way (eg. that we're trying to avoid running `. /etc/clearwater/config` in the main context).